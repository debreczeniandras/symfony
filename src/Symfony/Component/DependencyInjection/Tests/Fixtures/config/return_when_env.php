<?php

use Symfony\Component\DependencyInjection\Tests\Fixtures\Bar;
use Symfony\Config\ServicesConfig;

return [
    'when@prod' => [
        'parameters' => [
            'foo_param' => 'bar_value',
        ],
        new ServicesConfig(
            defaults: [
                'public' => true,
            ],
            services: [
                Bar::class => null,
                'my_service' => [
                    'class' => Bar::class,
                    'arguments' => ['%foo_param%'],
                ],
            ]
        ),
    ],
];
