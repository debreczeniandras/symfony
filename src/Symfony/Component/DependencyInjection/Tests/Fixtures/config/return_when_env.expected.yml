parameters:
    foo_param: bar_value

services:
    service_container:
        class: Symfony\Component\DependencyInjection\ContainerInterface
        public: true
        synthetic: true
    Symfony\Component\DependencyInjection\Tests\Fixtures\Bar:
        class: Symfony\Component\DependencyInjection\Tests\Fixtures\Bar
        public: true
    my_service:
        class: Symfony\Component\DependencyInjection\Tests\Fixtures\Bar
        public: true
        arguments: [bar_value]
